crates/noirc_evaluator/src/ssa/optimizations.rs

```diff
@@ -98,6 +99,49 @@ fn evaluate_intrinsic(
             }
             unreachable!();
         }
+        builtin::Opcode::ToRadix(endian) => {
```

Comment:
You are not using a radix base here.

Also you can should use this pattern instead of mutating element: `element= if little_endian {...} else {...};`

Reply:
What you can do simply is convert to BigUint and then use the to_radix from BigUint.